exercises/ledger/Ledger.cs

```diff
@@ -5,7 +5,7 @@
 
 public class LedgerEntry
 {
-   public LedgerEntry(DateTime date, string desc, float chg)
+   public LedgerEntry(DateTime date, string desc, decimal chg)
```

Comment:
Nice! Could you also update the `Example.cs` implementation (which is in the same directory)?